src/main/java/picard/util/ScatterIntervalsByNs.java

```diff
@@ -34,19 +60,33 @@
 )
 @DocumentedFeature
 public class ScatterIntervalsByNs extends CommandLineProgram {
-    static final String USAGE_SUMMARY = "Writes an interval list based on splitting a reference by Ns.  ";
-    static final String USAGE_DETAILS = "This tool identifies positions in a reference where the bases are 'no-calls' and writes out an " +
-            "interval-list using the resulting coordinates. This can be used to create an interval list for " +
-            "whole genome sequence (WGS) for e.g. scatter-gather purposes, as an alternative to using fixed-length intervals. The number " +
-            "of contiguous nocalls that can be tolerated before creating a break is adjustable from the command line.<br />" +
+    static final String USAGE_SUMMARY = "Writes an interval list based on splitting a reference by Ns.";
+    static final String USAGE_DETAILS = "A Program for breaking up a reference into intervals of alternating regions of N and ACGT bases." +
+            "<p>" +
+            "<p>" +
+            "<p>" +
+            "Used for creating a broken-up interval list that can be used for scattering a variant-calling pipeline in a way that will not cause problems at the edges of the intervals. " +
+            "By using large enough N blocks (so that the tools will not be able to anchor on both sides) we can be assured that the results of scattering and gathering the variants with " +
+            "the resulting interval list will be the same as calling with one large region.\n" +
+            "<p>" +
+            "<h3>Input</h3>" +
+            "<p> - A reference file to use for creating the intervals (needs to have index and dictionary next to it.)</p>" +
+            "<p> - Which type of intervals to emit in the output (Ns only, ACGT only or both)" +
+            "<p> - An integer indicating the largest number of Ns in a contiguous block that will be \"tolerated\" and not converted into an N block.</p>" +
+            "<p>" +
+            "<h3>Output</h3>" +
+            "<p> - An interval list (with a sam header) where the names of the intervals are labeled (either N-block or ACGT-block) to indicate what type of block they define. </p>" +
+            "<p>" +
+            "<p>" +
             "<h4>Usage example:</h4>" +
+            "<h3>Create an interval list of intervals that do not contain any N blocks for use with haplotype caller on short reads</h3>" +
             "<pre>" +
-            "java -jar picard.jar ScatterIntervalsByNs \\<br />" +
-            "      R=reference_sequence.fasta \\<br />" +
-            "      OT=BOTH \\<br />" +
-            "      O=output.interval_list" +
-            "</pre>" +
-            "<hr />";
+            "java -jar picard.jar ScatterIntervalsByNs \\\\<br />\n" +
```

Comment:
Not sure how this is rendered but I guess it should follow the same principle ... within the pre new lines and blanks are preserverved so there is no need to br's here either.